variables.tf

```
@@ -58,3 +58,13 @@ variable "region" {
   description = "The bucket's region location"
   default     = "us-central1"
 }
+
+variable "subnetwork" {
+  description = "The subnetwork to which VMs will be assigned. Should be of the form \"regions/REGION/subnetworks/SUBNETWORK\"."
+  default     = ""
+}
+
+variable "network" {
+  description = "The network to which VMs will be assigned. If it is not provided, \"default\" will be used."
+  default     = ""
```

Based on the description:

```suggestion
  default     = "default"
```

---

Fixed in cf5829ab3b9407dbfbac16814b17d744f87c59ba

---

This was missed in cf5829a, but looking at it now, what you have seems fine as `machine_type` is optional. Also, I don't know where I came up with the value of `"default"`. 😅